src/pages/services/aem-universal-editor/api/commons/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Universal Editor Extensibility
+description: Learn how to customize Universal Editor
+contributors:
+  - https://github.com/AdobeDocs/uix
+---
+
+# Common Concepts in Creating Extensions
+
+Understand the fundamentals required to develop an extension for the Universal Editor.
+
+## Extension Point
+
+Universal editor has an `universal-editor/ui/1` [extension point](https://developer.adobe.com/app-builder/docs/guides/extensions/) that allows you to extend its functionality.
+To declare it to be used by your extension, you need to add the following configuration to your `app.config.yaml` at the
+root of your extension:
+
+```yaml
+extensions:
+  universal-editor/ui/1:
+    $include: src/universal-editor-ui-1/ext.config.yaml
+```
+Here is an example of `ext.config.yaml` file:
+
+```yaml
+operations:
+  view:
+    - type: web
+      impl: index.html
+actions: actions
+web: web-src
+```
+
+## Extension Registration
+
+Interaction between UI Extension and Universal Editor starts with the initialization process that includes extension's
+capabilities registration so Universal Editor knows when to invoke the extension. Registration is done by `register` 
+method provided by `@adobe/uix-guest` library. This asynchronous method takes single object that describes extension 
+and returns object representing connection to the Universal Editor.
+
+Method `register` should be invoked after extension initialization page is loaded.
+
+Extension registration data must include:
+
+- `id` - string with random extension identifier. This identifier useful for debugging of interaction between Universal
+Editor and extension and needed if extension provides custom UI.
+- `methods` - objects with extension code exposed to the Universal Editor console. All methods are grouped into 
+namespaces that represents extension points provided by the Universal Editor.
+Currently, the following **namespaces** are available:
+- _headerMenu_, that allows to add buttons to the header of the Universal Editor;
+- _rightPanel_, that allows to add custom content under the rails to the right panel of the Universal Editor;
+- _canvas_, that allows to add custom renderer for data types
```

Comment:
Shouldn't this be another level of bullet points.

Reply:
fixed